src/otx/cli/utils/jsonargparse.py

```diff
@@ -0,0 +1,140 @@
+"""Functions related to jsonargparse."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import ast
+from pathlib import Path
+from typing import Any, TypeVar
+
+import docstring_parser
+from jsonargparse import ActionConfigFile, ArgumentParser, Namespace, dict_to_namespace
+
+
+def get_short_docstring(component: TypeVar) -> str | None:
+    """Get the short description from the docstring.
+
+    Args:
+        component (object): The component to get the docstring from
+
+    Returns:
+        Optional[str]: The short description
+    """
+    if component.__doc__ is None:
+        return None
```

Comment:
Can we `return ""` to make a single return type?

Reply:
Thanks! Done - https://github.com/openvinotoolkit/training_extensions/pull/2805/commits/1a254d9d1306b9389b89a5594eefa8faf43f2e38